src/libraries/System.Text.Json/tests/Serialization/OptionsTests.cs

```diff
@@ -538,30 +548,6 @@ public static void PredefinedSerializerOptions_UnhandledDefaults(int enumValue)
             Assert.Throws<ArgumentOutOfRangeException>(() => new JsonSerializerOptions(outOfRangeSerializerDefaults));
         }
 
-        private static JsonSerializerOptions CreateOptionsInstance()
-        {
-            var options = new JsonSerializerOptions
-            {
-                AllowTrailingCommas = true,
-                DefaultBufferSize = 20,
-                DictionaryKeyPolicy = JsonNamingPolicy.CamelCase,
-                Encoder = JavaScriptEncoder.Default,
-                IgnoreNullValues = true,
-                IgnoreReadOnlyProperties = true,
-                MaxDepth = 32,
-                PropertyNameCaseInsensitive = true,
-                PropertyNamingPolicy = new SimpleSnakeCasePolicy(),
-                ReadCommentHandling = JsonCommentHandling.Disallow,
-                ReferenceHandling = ReferenceHandling.Default,
-                WriteIndented = true,
-            };
-
-            options.Converters.Add(new JsonStringEnumConverter());
-            options.Converters.Add(new ConverterForInt32());
-
-            return options;
-        }
-
```

Comment:
Is this unused code?

Reply:
Yes, it's unused. It should have been removed in https://github.com/dotnet/runtime/pull/36322.